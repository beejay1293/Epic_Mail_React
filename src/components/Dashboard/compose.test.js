import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Compose from './compose';
describe('Compose message button', () => {
  it('should render correctly the compose message button', () => {
    const ComposeBtn = shallow(<Compose />);
  
    expect(ComposeBtn).toMatchSnapshot();
  });
});
