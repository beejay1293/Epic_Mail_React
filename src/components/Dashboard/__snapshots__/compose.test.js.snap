// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compose message button should render correctly the compose message button 1`] = `
<div
  className="compose__message_wrapper"
>
  <div
    className="compose__message_btn"
  >
    COMPOSE
    <div
      className="plus__icon"
    >
      <i
        className="fas fa-plus"
      />
    </div>
  </div>
</div>
`;
